x/auth/types/genesis.go

```diff
@@ -33,6 +33,9 @@ func NewGenesisState(params Params, accounts GenesisAccounts) *GenesisState {
 func (g GenesisState) UnpackInterfaces(unpacker types.AnyUnpacker) error {
 	for _, any := range g.Accounts {
 		var account GenesisAccount
+		if any.TypeUrl == "/ibc.applications.interchain_accounts.v1.InterchainAccount" {
```

Comment:
woaaa, why is this necessary?

Reply:
Also might be better to call MsgType on the type directly instead of a hard-coded string in case it changes.

Reply:
IRT the type vs hard code, this is no longer needed in v12 so even if this changed in the future it would not affect this AFAIU!